Core/FitParameter.hpp

```diff
@@ -20,14 +20,51 @@
 #include <boost/property_tree/ptree.hpp>
 #include <boost/serialization/utility.hpp>
 #include <boost/optional.hpp>
+#include <boost/algorithm/string/predicate.hpp>
 
 #include "Core/Parameter.hpp"
 #include "Core/Exceptions.hpp"
 #include "Core/Logging.hpp"
 
+struct BoolTranslator {
+  typedef std::string internal_type;
+  typedef bool external_type;
+
+  // Converts a string to bool
+  boost::optional<external_type> get_value(const internal_type &str) {
+    if (!str.empty()) {
+    	using boost::algorithm::iequals;
+      if (iequals(str, "true") || iequals(str, "yes") || str == "1")
```

Comment:
We could add "True" here

Reply:
That is already included. It uses iequals, which is a case insensitive compare. So TRUE True TRue true YES Yes yes etc... and 1 all works